litellm/proxy/proxy_server.py

```diff
@@ -4563,13 +4586,13 @@ async def audio_transcriptions(
         cache_key = hidden_params.get("cache_key", None) or ""
         api_base = hidden_params.get("api_base", None) or ""
 
-        fastapi_response.headers["x-litellm-model-id"] = model_id
-        fastapi_response.headers["x-litellm-cache-key"] = cache_key
-        fastapi_response.headers["x-litellm-model-api-base"] = api_base
-        fastapi_response.headers["x-litellm-version"] = version
-        fastapi_response.headers["x-litellm-model-region"] = (
-            user_api_key_dict.allowed_model_region or ""
-        )
+        fastapi_response.headers.update(get_custom_headers(
+            model_id=model_id,
+            cache_key=cache_key,
+            api_base=api_base,
+            version=version,
+            model_region=user_api_key_dict.allowed_model_region
```

Comment:
shouldn't this be 
`user_api_key_dict.allowed_model_region or ""` like we used to before

Reply:
can you make it 

`getattr(user_api_key_dict, "allowed_model_region", "")`

Reply:
Please use `getattr(user_api_key_dict, "allowed_model_region", "")` here 

It's safely accessing user_api_key_dict

Reply:
sure

Reply:
Done with the changes, here are test:
<img width="1231" alt="image" src="https://github.com/BerriAI/litellm/assets/4735661/a8cc8cfe-d86f-4e81-93c3-62858bfdbc77">
<img width="545" alt="image" src="https://github.com/BerriAI/litellm/assets/4735661/01e3f764-7e4a-457f-a927-9134284e779f">